cmd/operator-sdk/alpha/olm/down.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
+	"fmt"
+	"log"
+
+	olmoperator "github.com/operator-framework/operator-sdk/internal/olm/operator"
+
+	"github.com/spf13/cobra"
+)
+
+func NewDownCmd() *cobra.Command {
+	c := &olmoperator.OLMCmd{}
+	cmd := &cobra.Command{
+		Use:   "down",
+		Short: "Tear down your operator with Operator Lifecycle Manager",
```

Comment:
Besides `Tear down` be a common nomenclature I'd prefer maybe deploy / undeploy since we have the deploy dir in the project. But it is fine .. please feel free to keep it if it please you.

Reply:
We already have `operator-sdk up local` that users are familiar with. The intention here is to use the same familiar nomenclature.

Reply:
+1 .. let's try keep a concise nomenclature across the project it make the things more intuitive and easier to work with.